vendor/github.com/modern-go/reflect2/type_map.go

```diff
@@ -1,17 +1,13 @@
+// +build !gccgo
```

Comment:
what are the implications of this for downstream consumers?

Reply:
This got merged via https://github.com/modern-go/reflect2/pull/10 to avoid building it on gccgo, and I'm not sure if anyone is really still using gccgo version, I remember we(ppc64le) were using gccgo during the initial days but not anymore.